plugin/metric/collector.go

```diff
@@ -0,0 +1,10 @@
+package metric
```

Comment:
The collector doesn't seem to belong to the metric package. Metric is responsible for metric reporting, while the collection is with the core package instead of the plugin.